atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -1484,6 +1486,32 @@ public void commitThrowsIfRolledBackAtCommitTime_expiredLocks() {
                 .hasLocksExpired(1);
     }
 
+    @Test
+    public void commitThrowsIfLockAcquisitionFails() {
+        final Cell cell = Cell.create(PtBytes.toBytes("row1"), PtBytes.toBytes("column1"));
+
+        TimelockService timelockServiceMock = mock(TimelockService.class);
+        // Let request to acquire commit locks time out
```

Comment:
I don't think the comment adds much here! Let's remove it, but it's good to see that you're thinking about readability